NEWS.md

```diff
@@ -1,5 +1,11 @@
 # Packrat 0.9.0 (UNRELEASED)
 
+- Packrat now masks environment variables commonly used for Git service account
+  authentication from subprocesses used to install packages. This behavior can
+  be disabled by setting the option `packrat.unmask.git.service.vars` to `TRUE`.
```

Comment:
What about using `packrat.mask.git.service.vars` with a default of `TRUE` and using `FALSE` to stop variable masking. This option feels more like "disable the feature" rather than "enable the absence of the feature".

Reply:
Agreed. I went back and forth on which way to go, and with the distance of a weekend, your suggestion seems much clearer.